newIDE/app/src/ResourcesList/ResourcePreview/ImagePreview.js

```diff
@@ -179,11 +177,11 @@ const ImagePreview = ({
         const imageLoaded = !!imageWidth && !!imageHeight && !errored;
 
         // Once the container is loaded, adapt the zoom to the image size.
-        if (!isResizeObserverReady && containerLoaded) {
+        if (!isResizeObserverReady.current && containerLoaded) {
           if (!initialZoom) {
             adaptZoomToImage(containerHeight, containerWidth);
           }
-          setIsResizeObserverReady(true);
+          isResizeObserverReady.current = true;
```

Comment:
are you sure this works well? as the state change would trigger a rerender, but not the ref change.
(Just thinking out loud, to ensure you've tested it well)

Reply:
Well, it does not seem to have changed something. I fixed this because there was a React error stating that we can't use a setState in the render method of a component.

Reply:
But I'll revert the commit, I'll have a look at this in another PR